rollup.config.js

```diff
@@ -18,42 +18,49 @@ const tsOpts = {
 };
 
 const globals = {
-    react: 'automat.react',
-    emotion: 'automat.emotion', // TODO remove this dependency
-    '@emotion/core': 'automat.emotionCore',
-    'emotion-theming': 'automat.emotionTheming',
+    react: 'guardian.automat.react',
```

Comment:
Oh - how come `guardian.` got prepended to these? Is this related to a change on the platform?

Reply:
Yeah, it came from a suggestion on Nic's PR to use the guardian namespace.